lms/djangoapps/learner_home/views.py

```diff
@@ -430,9 +429,16 @@ def serialize_learner_home_data(data, context):
     return LearnerDashboardSerializer(data, context=context).data
 
 
-class InitializeView(RetrieveAPIView):  # pylint: disable=unused-argument
+class InitializeView(APIView):  # pylint: disable=unused-argument
     """List of courses a user is enrolled in or entitled to"""
 
+    authentication_classes = (
+        JwtAuthentication,
```

Comment:
JwTAuth? Does this view use a jwt token?

Reply:
I think jwt-related things can go

Reply:
Unclear, but that's what is on most of our APIs . Was just following the pattern. Should I remove?

Reply:
Know what? It won't hurt. Just leave it

Reply:
It's something I'm like 90% sure isn't necessary, but I don't want to break something if that 10% is there